plugins/CoreHome/angularjs/multipairfield/multipairfield.controller.js

```diff
@@ -34,8 +34,16 @@
             $scope.field2.templateFile = getTemplate($scope.field2);
         }
 
+        if ($scope.field3 && !$scope.field3.templateFile) {
+            $scope.field3.templateFile = getTemplate($scope.field3);
+        }
+
+        if ($scope.field4 && !$scope.field4.templateFile) {
+            $scope.field4.templateFile = getTemplate($scope.field4);
+        }
+
         var self = this;
-        $scope.$watch('formValue', function () {
+        $scope.$watch('formValue', function () {;
```

Comment:
The semicolon might be there by accident?

Reply:
```suggestion
        $scope.$watch('formValue', function () {
```